src/rdb_protocol/error.hpp

```diff
@@ -120,6 +120,9 @@ class bt_rcheckable_t : public rcheckable_t {
     backtrace_id_t bt;
 };
 
+//Helper function for formatting array over size error message.
```

Comment:
For consistency with other comments, could you add a space between `//` and `Helper`?